src/introspector/dialects/postgres/date-parser.ts

```diff
@@ -0,0 +1,6 @@
+export const enum DateParser {
+  STRING = 'string',
+  TIMESTAMP = 'timestamp'
```

Comment:
Wasn't sure if it made sense to call this `TIMESTAMP` or `DATE`, but I went with `TIMESTAMP` to match the existing generated types.